### DISCUSSION TITLE
one kong as a proxy for another kong

### DISCUSSION THREAD

**Discussion by geniusquirtle** [Kong/kong#10362] [Help]
it works well for get, but there is a problem for post request. The following is a description of the problem。

2023/02/24 00:55:40 [warn] 48#0: *6636131 using uninitialized "kong_proxy_mode" variable while logging request, client: 192.168.1.68, server: kong, request: "POST /authority/base_cas/captcha HTTP/1.1", host: "host50.service.consul:8443", referrer: "https://192.168.1.68:8443/html/login"
2023/02/24 00:55:40 [warn] 48#0: *6636131 [lua] reports.lua:80: log(): [reports] could not determine log suffix (scheme=http, proxy_mode=) while logging request, client: 192.168.1.68, server: kong, request: "POST /authority/base_cas/captcha HTTP/1.1", host: "host50.service.consul:8443", referrer: "https://192.168.1.68:8443/html/login"
192.168.1.68 - elastic [24/Feb/2023:00:55:40 +0000] "POST /authority/base_cas/captcha HTTP/1.1" 400 622 "https://192.168.1.68:8443/html/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36"

**Comment by flrgh** [comment]
hi @geniusquirtle :wave: 

Can you please include some more details about your setup/environment/topology? It's not exactly clear what `one kong as a proxy for another kong` means in this context. Which version(s) of Kong are you running? If you are using a custom nginx template or any injected nginx conf directives, this will be important to include as well.

**Reply by geniusquirtle** [reply]
I have tested two versions  2.7.1 and 3.1.1. There are the same results. I use the default config without any custom nginx template. one uses consul as KONG_DNS_RESOLVER.

**Reply by flrgh** [reply]
Thank you for the extra info. Can you include any kind of minimal [kong.yml](https://docs.konghq.com/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/#declarative-configuration-format) and docker-compose file that we can use to reproduce the issue?
